src/plugins_3rdparty/primer3/transl/russian.ts

```diff
@@ -816,1212 +846,1310 @@
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Используйте термодинамические модели для расчета склонности олигонуклеотидов к образованию шпилек и димеров.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
-        <location filename="../src/Primer3Dialog.ui" line="1596"/>
+        <location filename="../src/Primer3Dialog.ui" line="1697"/>
         <source>Use Thermodynamic Oligo Alignment</source>
-        <translation type="unfinished"></translation>
+        <translation>Использовать Термодинамическое Олигонуклеотидное Выравнивание</translation>
     </message>
     <message>
-        <location filename="../src/Primer3Dialog.ui" line="1608"/>
+        <location filename="../src/Primer3Dialog.ui" line="1709"/>
+        <location filename="../src/Primer3Dialog.ui" line="1848"/>
         <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Same as&lt;span style=&quot; font-weight:700;&quot;&gt; Max 3&apos; Self Complementarity&lt;/span&gt; but is based on thermodynamical approach - the stability of structure is analyzed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;То же, что и &lt;span style=&quot; font-weight:700;&quot;&gt; Максимальная Самокомплементарность на 3&apos;&lt;/span&gt;, но основанная на термодинамическом подходе - анализируется стабильность структуры.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
-        <location filename="../src/Primer3Dialog.ui" line="1650"/>
+        <location filename="../src/Primer3Dialog.ui" line="1751"/>
+        <location filename="../src/Primer3Dialog.ui" line="2007"/>
         <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Same as &lt;span style=&quot; font-weight:700;&quot;&gt;Max 3&apos; Pair Complementarity&lt;/span&gt; but for calculating the score (melting temperature of structure) thermodynamical approach is used.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;То же, что и &lt;span style=&quot; font-weight:700;&quot;&gt;Максимальная Парная Комплементарность на 3&apos;&lt;/span&gt; но для расчета значения (температуры плавления структуры) используется термодинамический подход&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
-        <location filename="../src/Primer3Dialog.ui" line="1666"/>
+        <location filename="../src/Primer3Dialog.ui" line="1770"/>
         <source>Max 3&apos; Pair Complementarity</source>
-        <translation type="unfinished"></translation>
+        <translation>Максимальная Парная Комплементарность на 3&apos;</translation>
```

Comment:
Please do not translate into Russian with capital letters even if you use them in English. This is a common style for UGENE translations  (please update all such translations in this PR).

I would also suggest using `Макс.` instead of `Максимальная` to make the source and the translation have a close length.

Reply:
OK